melange-build-pkg/action.yaml

```diff
@@ -75,7 +75,7 @@ runs:
         [ -n '${{ inputs.repository-append }}' ] && repoarg="--repository-append ${{ inputs.repository-append }}"
         [ -n '${{ inputs.keyring-append }}' ] && keyringarg="--keyring-append ${{ inputs.keyring-append }}"
         [ -n '${{ inputs.workspace-dir }}' ] && workspacearg="--workspace-dir ${{ inputs.workspace-dir }}"
-        [ -n '${{ inputs.template }}' ] && templatearg="--template ${{ inputs.template }}"
+        [ -n '${{ inputs.template }}' ] && templatearg="--template '${{ inputs.template }}'"
```

Comment:
i think the issue is the `[ -n '...' ]` part actually.

Reply:
so i was passing in `"'{\"Version\": \"${{ matrix.nginx-version }}\" }'"`  as my initial input & i think the single quotes in the input were messing up the `[ -n '...' ]` part.

i think with this change i can remove the single quotes from the input and that shouldn't mess up `[ -n '...' ]` anymore?

Reply:
i'm not great at bash so i could be totally wrong about that 😅

Reply:
yeah if the input has changed it is probably fine, but this seems fragile.

I recently learned that github actions has `${{ foo && "..." }}` substitution syntax.  we can probably build something more robust with that?

Reply:
looked through the [docs](https://docs.github.com/en/actions/learn-github-actions/expressions#literals) & it seems like it still expects strings to be wrapped in single quotes

>  You don't need to enclose strings in ${{ and }}. However, if you do, you must use single quotes (') around the string. To use a literal single quote, escape the literal single quote using an additional single quote (''). Wrapping with double quotes (") will throw an error.

i'd prefer smth more robust as well. for now, avoiding single quotes in the input might be a temporary fix?